backend/oasst_backend/api/v1/tasks.py

```diff
@@ -155,7 +155,7 @@ async def interaction_tx(session: deps.Session):
         tm = TreeManager(session, pr)
         ur = UserRepository(session, api_client)
         task = await tm.handle_interaction(interaction)
-        if type(task) is protocol_schema.TaskDone:
+        if isinstance(task, protocol_schema.TaskDone):
```

Comment:
`isinstance()` and `type() is`  are two different things, these checks were exact type checks. The rewritten form is longer than the old.